### PR TITLE
feat: Add onboarding wizard instructions for Svelte SDK

### PR DESCRIPTION
**Pull request by Lms24**
This PR adds the onboarding wizard page for the Svelte SDK.

ref: https://github.com/getsentry/sentry-javascript/issues/5671

### CODE REVIEW

File: src/wizard/javascript/svelte.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+});
+
+const app = new App({
+  target: document.getElementById("app"),
+});
+
+export default app;
+```
+
+Once you've verified the library is initialized properly and sent a test event, consider visiting our [complete Svelte docs](https://docs.sentry.io/platforms/javascript/guides/svelte/). There, you'll find additional instructions for configuring the Svelte SDK.
```
Comment (imatwawana) [inline src/wizard/javascript/svelte.md:46]: I've suggested this edit to make it a bit more concise.

```suggestion
Once you've verified the library is initialized properly and you've sent a test event, check out our [complete Svelte docs](https://docs.sentry.io/platforms/javascript/guides/svelte/) for additional configuration instructions.
```

Comment (imatwawana) [inline src/wizard/javascript/svelte.md:46]: Shouldn't we have a test snippet in the wizard?

Comment (Lms24) [reply src/wizard/javascript/svelte.md:46]: Much better, thanks!

Comment (Lms24) [reply src/wizard/javascript/svelte.md]: Oh yes, for sure. I think we should also update the Remix wizard with a test snippet as it's missing there as well.

**Review by imatwawana — APPROVED** [review]
LGTM!

**Pull request merged**